### PR TITLE
Gbear605 typo fixes

### PR DESCRIPTION
**Pull request by gbear605**
Fix English typos detailed in #5631

### CODE REVIEW

File: common/locales/en/settings.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "usernameSuccess": "Login Name successfully changed",
     "emailSuccess": "Email successfully changed",
     "detachFacebook": "De-register Facebook",
-    "detachedFacebook": "Successully removed Facebook from your account",
+    "detachedFacebook": "Successfully removed Facebook from your account",
     "addedLocalAuth": "Successully added local authentication",
```
Comment (crookedneighbor) [inline common/locales/en/settings.json]: Successfully is also misspelled on this line.

Comment (crookedneighbor) [inline common/locales/en/settings.json]: I know it wasn't in the original issue, but we should fix it while we're doing this.

Comment (gbear605) [inline common/locales/en/settings.json]: Fixed in the most recent commit

**Comment by crookedneighbor** [comment]
Seems reasonable to me :+1:

**Comment by crookedneighbor** [comment]
I decided to revert the em dash changes. It does not seem important enough to me to make the translators re-translate the strings for those changes.

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by gbear605** [comment]
2893ce69-f5df-41a5-8d4f-59fe286660cf

**Comment by crookedneighbor** [comment]
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Comment by ghost** [comment]
@crookedneighbor regarding your comment

> I decided to revert the em dash changes. It does not seem important enough to me to make the translators re-translate the strings for those changes.

I just want to mention that if you change source (English) string but keep the same string-key, Transifex keeps the history of the string which lets translators just reuse previous translation.

I've closed the issues asking for the hyphen -> em-dash change. For possible future reference the reported lines can be found in the commits above or with the help of Transifex filtering: [link](https://www.transifex.com/habitrpg/habitrpg/translate/#ru/$/21450415?comment=dash)

**Pull request merged**